.eleventy.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function (eleventyConfig) {
```

Comment:
ESLint should be complaining about lack of `'use strict'` here.  maybe it's misconfigured

Reply:
I'm going to guess it's not reading the file at all, because it should also complain about double-quotes.

Reply:
Fixed